app/components/ModalWrapper/index.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+export const Modal = ({ handleClose, show, children }) => {
+  const showHideClassName = show ? "modal display-block" : "modal display-none";
```

Comment:
no need to switch the css classes.
Instead either we render or no render the component.